service/api.go

```diff
@@ -88,15 +88,15 @@ func MustHaveProject(r *http.Request) *model.Project {
 }
 
 // requireUserToggleable wraps gimlet.NewRequireAuthHandler and checks that
-// a user is generally authenticated if the PartialRouteAuthDisabled flag is set.
+// a user is generally authenticated if the RestRouteRequireAuthDisabled flag is set.
```

Comment:
This comment phrasing implies the opposite of what it actually does - if the `RestRouteRequireAuthDisabled` flag is true (i.e. set), then it will skip the auth requirement. Maybe phrase this as `if the RestRouteRequireAuthDisabled flag is false`?

Reply:
The naming schema for our admin flags messed me up here. Since they all have "Disabled" in their variable name, yet on the actual admin page [Enabled](https://evergreen.mongodb.com/admin) really means the value is false (which is all flags default value) under the hood is unintuitive. I'm considering if a ticket to make this easier to understand would be helpful.

I think the real problem is I named the variables incorrectly. I've now changed them from `RestRouteRequireAuthDisabled ` --> `RestRoutePartialAuthDisabled `, to make the variable name actually do what it says it does when reading the code. The value will initially be false, meaning the [admin page](https://evergreen.mongodb.com/admin) will say that "partial auth" will be enabled, which means the code `if flags.RestRoutePartialAuthDisabled { do_auth() }` makes sense